metricbeat/mb/module/wrapper.go

```diff
@@ -185,6 +186,11 @@ func (mw *Wrapper) Hash() uint64 {
 	return mw.configHash
 }
 
+// MetricSets return the list of metricsets of the module
+func (mw *Wrapper) MetricSets() []*metricSetWrapper {
```

Comment:
Can this return `[]mb.MetricSet` instead? (Returning an unexported type from an exported method can be annoying to users.)